eng/bomgenerator/includes/com.azure.resourcemanager.txt

```diff
@@ -12,7 +12,6 @@ azure-resourcemanager-dns
 azure-resourcemanager-eventhubs
 azure-resourcemanager-keyvault
 azure-resourcemanager-monitor
-azure-resourcemanager-msi
```

Comment:
Do we need to change this file?

Reply:
Good catch! No, we don't. Thought I reverted this line..

Reply:
Also check the file above.